calicoctl.py

```diff
@@ -142,7 +146,11 @@ def container_add(container_name, ip):
     ip = IPAddress(ip)
     version = "v%s" % ip.version
     pools = client.get_ip_pools(version)
-    if not any([ip in pool for pool in pools]):
+    for candidate_pool in pools:
+        if ip in candidate_pool:
+            pool = candidate_pool
+
+    if pool is None:
         print "%s is not in any configured pools" % ip
```

Comment:
I think this needs a `pool = None` line before the for loop. Otherwise, if we don’t find the IP in any candidate pools, the value of `pool` will be unassigned and we’ll throw a NameError when we hit the if statement.